### PR TITLE
* Add missing case to `PaletteMicrosoft365SilverDarkMode.cs`

### PR DESCRIPTION
**Pull request by Wagnerp**
* Add missing case to `PaletteMicrosoft365SilverDarkMode.cs`
* Some Ribbon designer implementations

image

### CODE REVIEW

File: Source/Krypton Components/Krypton.Ribbon/Palette/RibbonAppButton.cs [hunk 1]
```diff
@@ -437,7 +437,7 @@ public bool FormCloseBoxVisible
                     if (_ribbon.CaptionArea != null)
                     {
                         _ribbon.CaptionArea.KryptonForm!.CloseBox = value;
-                        _ribbon.CaptionArea.PerformFormChromeCheck();
+                        _ribbon.CaptionArea?.PerformFormChromeCheck();
```
Comment (Smurf-IV) [inline Source/Krypton Components/Krypton.Ribbon/Palette/RibbonAppButton.cs:440]: Why was this change necessary, as the `CaptionArea` is used on the previous line "Without" the null check.

**Review by Smurf-IV — CHANGES_REQUESTED** [review]

File: Source/Krypton Components/Krypton.Ribbon/Palette/RibbonAppButton.cs (hunk 1 above)
Comment (tobitege) [reply Source/Krypton Components/Krypton.Ribbon/Palette/RibbonAppButton.cs:440]: Actually the "KryptonForm" in line 439 should be checked for null at that location, not the CaptionArea as the outer "if" already assures that.

Comment (Wagnerp) [reply Source/Krypton Components/Krypton.Ribbon/Palette/RibbonAppButton.cs]: Undone, but what's causing #1308?

Comment (tobitege) [reply Source/Krypton Components/Krypton.Ribbon/Palette/RibbonAppButton.cs]: > Undone, but what's causing #1308?

Exactly what I wrote in my above comment and in the issue report:
**_ribbon.CaptionArea.KryptonForm** is null, thus needs to be checked for null before assignment
